backend/controller/paymentController.js

```diff
@@ -154,9 +215,23 @@ const createPlan = asyncHandler(async (req, res, next) => {
 
   const result = await planInfo.save();
 
-  return res.status(200).json({ success: true, data: result });
+  return res.status(200).json({
```

Comment:
This is for creating a plan, so the status should be 201

Reply:
added 201